lib/src/animated_map_controller.dart

```diff
@@ -100,15 +100,15 @@ class AnimatedMapController {
       );
     }
 
-    final effectiveDest = dest ?? mapController.center;
-    final effectiveZoom = zoom ?? mapController.zoom;
+    final effectiveDest = dest ?? mapController.camera.center;
+    final effectiveZoom = zoom ?? mapController.camera.zoom;
     final effectiveRotation = rotation ?? this.rotation;
     final latLngTween = LatLngTween(
-      begin: mapController.center,
+      begin: mapController.camera.center,
       end: effectiveDest,
     );
     final zoomTween = Tween<double>(
-      begin: mapController.zoom,
+      begin: mapController.camera.zoom,
```

Comment:
Maybe directly create a variable that would hold `mapController.camera` to reduce verbosity:
```dart
final camera = mapController.camera;

final effectiveDest = dest ?? camera.center;
final effectiveZoom = zoom ?? camera.zoom;
```